packages/mux-player/src/helpers.ts

```diff
@@ -112,3 +112,99 @@ export const isInLiveWindow = (el: MuxPlayerElement) => {
   }
   return false;
 };
+
+export class AttributeTokenList {
```

Comment:
suggestion(non-blocker): Consider moving this into its own module, since it's an implementation of a generic class.

Reply:
this seems like it'll be useful when we'd want a `hotkeys` list like controlslist